common/transform/build.gradle

```diff
@@ -1,13 +1,15 @@
+apply from: project(":").file('gradle/java.gradle')
+
 dependencies {
-    compile project(':common:rest-java-model')
+    implementation project(':common:rest-java-model')
 
-    compile group: 'ai.h2o', name: 'mojo2-runtime-api'
-    compile group: 'ai.h2o', name: 'mojo2-runtime-impl'
+    implementation group: 'ai.h2o', name: 'mojo2-runtime-api'
+    implementation group: 'ai.h2o', name: 'mojo2-runtime-impl'
 
```

Comment:
ultra nit: There is a newline between `implementation group:...` and `testImplementation group:...` blocks here, while other build files (kbd, local-rest-scorer) do not.

I have no issue with this whatsoever, just pointing out in case you prefer one way or another.

Reply:
Thanks @orendain. So I unified this now as follows:
- non-test deps
- newline
- test deps
Tbh, I don't care too much as long as the different types are not mixed. If you prefer some other way, I am happy to do that (I understand it is not really important, but it does not really cost anything either, so why not :)).